### ISSUE TITLE
deprecate argocd-notifications docs site

### ISSUE THREAD

**Issue by asaf400** [argoproj/argo-cd#15184]
PLEASE !

Add a deprecation notice AT THE TOP of the site, at least 15% of the page,
that the docs for https://argocd-notifications.readthedocs.io/en/stable/ are for the old and deprecated project
[argocd-notifications](https://github.com/argoproj-labs/argocd-notifications/)

and that now the features have been implemented and are here:
https://argo-cd.readthedocs.io/en/latest/operator-manual/notifications/

any google search for "argocd notifications" gets you to that old doc pages,
an unaware user should not go to the github of the project in order to see that it is no longer supported!

P.S I would have created a PR for this, but the repo is archived..

**Comment by crenshaw-dev** [comment]
Yep. Trying to get some help on this: https://cloud-native.slack.com/archives/C020XM04CUW/p1692801796863639

**Comment by mztnnrt** [comment]
We have same problem. For example, below way is not working for our environment.

- [Helm v3 Getting Started](https://argocd-notifications.readthedocs.io/en/stable/#helm-v3-getting-started)
- [Kustomize Getting Started](https://argocd-notifications.readthedocs.io/en/stable/#kustomize-getting-started)

**Comment by thiagowfx** [comment]
This still causes confusion to my teammates. :(

I am happy to help, however the only way to make this change is by an admin of the archived repo.

**Comment by crenshaw-dev** [comment]
> DO SOMETHING ABOUT IT ALREADY

> an unmaintained mess of a project

@asaf400 use shout-case on an Argo CD issue or make rude remarks again, and I'll delete the comment. Do so again after that, and I'll block you from the repo.

We're a finite team with finite resources, but I have the resources to remove toxic behavior.
